crates/ruma-common/src/identifiers.rs

```diff
@@ -33,6 +33,18 @@ pub use self::{
 #[doc(inline)]
 pub use ruma_identifiers_validation::error::Error;
 
+/// Provides owned variants for opaque identifiers.
+pub mod owned {
```

Comment:
I don't think these should be in a separate sub-module.

Reply:
It felt weird mixing them inbetween the actual identifiers, but alright 👍